vignettes/usage_fixed_design.Rmd

```diff
@@ -9,7 +9,7 @@ output:
     code_folding: hide
     highlight: "textmate"
     css: "custom.css"
-bibliography: gsDesign.bib
+bibliography: ggsd.bib
```

Comment:
Should this be `gsDesign2.bib` instead?

Reply:
gsDesign.bib and ggsd.bib share a lot of bib in common. Since Keaven initially called it ggsd.bib, I keep it as it is there. Let's get back later after Keaven approves the vignettes list.